src/utils/color-utils.js

```diff
@@ -60,6 +60,15 @@ const colorUtils = {
     const current = getCurrentTheme(Theme);
     return (current && current.properties.fontFamily) || 'inherit';
   },
+  lightenOrDarkenColor(color, percent) {
```

Comment:
Use a more descriptive name, `getFadedColor` or something. At least something with get, since it returns a color.